test/internal/scorecard.go

```diff
@@ -79,3 +79,17 @@ func (tc TestContext) AddScorecardCustomPatchFile() error {
 	}
 	return nil
 }
+
+func (tc TestContext) AddScorecardDevImage() {
```

Comment:
IMO we are not adding but replacing so I think that something such as:
 
```suggestion
// ReplaceScorecardImagesForDev will replaces the scorecard images in the manifests per the dev tag which is built 
// in the CI based on the code changes made. 
func (tc TestContext) ReplaceScorecardImagesForDev() {
```

Reply:
You might be able to find a better text/comment. However, let's comment all exported funcs.

Reply:
Done